pkgs/by-name/bi/bitrise/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "bitrise";
+  version = "2.19.0";
+
+  src = fetchFromGitHub {
+    owner = "bitrise-io";
+    repo = "bitrise";
+    rev = version;
+    hash = "sha256-VjuDeRl/rqA7bdhn9REdxdjRon5WxHkFIveOYNpQqa8=";
+  };
+
+  # many tests rely on writable $HOME/.bitrise and require network access
+  doCheck = false;
```

Comment:
Is it feasible to selectively disable the network tests? Some tests are still better than none

Reply:
I already have a draft PR for that, but the list of ignored tests is quite long: https://github.com/NixOS/nixpkgs/pull/333648/files#diff-6ed507756f46cf318b766f9e189a33a2c87be833e2ca6458c13b6fa43ff714dcR18

I'd rather do the clean up and refactor of the tests in the repo itself (there are distinct integration tests already, I think these failing tests should be tagged as integration tests). But I'd prefer merging this first PR with `doCheck = false`, and then iterate on it.

Reply:
Ok, fine with me :+1: